docs/src/main/asciidoc/flyway.adoc

```diff
@@ -185,9 +233,17 @@ public class MigrationService {
     @Inject
     Flyway flyway; <1>
 
+    @Inject
+    @Named("flyway_users") <2>
+    Flyway flywayForUsers;
+
+    @Inject
+    @FlywayDataSource("inventory") <3>
```

Comment:
Let's make it the first proposal as I would prefer to recommend this usage.

Reply:
ok